differential-expression/README.md

```diff
@@ -1,25 +1,161 @@
+What you should find is that in the same folder `differential-expression/data/`, you now have a file named `GSE71270.gct`
+
+**Example 2**  
+After running the code chunk we showed above, let's try running the same thing a second time:
+
+```bash
+Rscript scripts/create_gct_file.R \
+ --file differential-expression/data/GSE71270.tsv
+```
+
+What you should see is an error message that says this:
+
+```
+differential-expression/data/GSE71270.gct already exists. Use '--rewrite' option if you want this file to be overwritten.
+```
+
+This is telling us that `create_gct_file.R` will not write over an already existing file unless we explicitly tell it to.
+
+**Example 3**  
+If we want to save over an already existing file, we need to use the `rewrite` option, just like the error message says.  
+Let's try that:
+
+```bash
+Rscript scripts/create_gct_file.R \
+ --file differential-expression/data/GSE71270.tsv \
+ --rewrite
+```
+
+This will rewrite over the file we made in *Example 1* but should give you a message to tell you it is doing so: `Overwriting file named differential-expression/data/GSE71270.gct`
+Also note that for bash commands, a `\` indicates that the command continues on the next line.
+Since we put `--rewrite` on the next line, we needed to add a `\` so that it knows that the command continues on the next line.
+
+**Example 4**  
+Lastly, if we would like to name the file something besides its original name, we can use the `--output` argument.
+Here let's save it directly to the `differential-expression` folder and call it something different.
+
+```bash
+Rscript scripts/create_gct_file.R \
+ --file differential-expression/data/GSE71270.tsv \
+ --output differential-expression/GSE71270_special_name.gct
+```
+
+Now if you should see a file called `differential-expression/GSE71270_special_name.gct`.
+
+#### Create a CLS format file
 
-For this [Gene Pattern notebook](https://notebook.genepattern.org/services/sharing/notebooks/22/preview/)
-you will need to have a gct and CLS file, so follow
-[these instructions](https://github.com/AlexsLemonade/refinebio-examples/blob/master/README.md)
-if you haven't created those files from your data yet.  
+CLS formatted files provide the sample groups or phenotype information and are necessary for performing gene expression differential analysis using GenePattern.
+If you've already created a GCT format file from your data, like is described above, you can create a a CLS format using GenePattern's online
+[CLSFileCreator](http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/ClsFileCreator/4)
 
-A general introduction to GenePattern notebooks is available [here](http://genepattern-notebook.org/public-notebooks/).
+*Now login into [GenePattern](https://cloud.genepattern.org/gp/pages/login.jsf), select a `Differential Expression` notebook, and follow the instructions to upload and analyze your newly created GCT and CLS files*
```

Comment:
This says notebook. It should say module. My comment earlier (https://github.com/AlexsLemonade/refinebio-examples/pull/66#discussion_r280101051) was not about changing this to notebook. In that review, I asked for us to transition to modules.

Reply:
Oh I see. Got confused.